platform/testing/end-to-end/cypress-best-practices-on-vsp.md

```diff
@@ -71,6 +72,14 @@ Cypress queues its commands instead of running them synchronously, so doing some
 
 Cypress has many third party [plugins](https://docs.cypress.io/plugins/) available. If you find yourself needing to do something that isn't natively supported, there may be a plugin for it.
 
+### Test retries
+
+When running tests on Jenkins, CircleCI, or locally in headless mode, Cypress will retry failing tests up to two times before marking the test as failed. This is meant to reduce the number of build failures from tests that fail occasionally.
+
+Tests should be written to be retry-able in order to use this Cypress feature effectively. Without proper test structure and implementation (set up, execution, teardown, etc.), tests can fail consistently upon retry.
```

Comment:
Is there anything specific about how tests should be written to be retry-able? I'm approaching this as someone completely new to our tech stack (or at least Cypress). I would be looking for a link to a separate section or a small code snippet to help me write my test in the intended manner.

Reply:
Thanks for the comment, I wasn't sure how much detail to go into in this document. I've updated the PR with a simple Google search test example.

Reply:
Thank you @ddzz . This has such an immediate impact, reading through the what and seeing the how right after.